src/CartBundle/Tests/Manager/CartManagerTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ *
+ */
+
+namespace Commercetools\Symfony\CartBundle\Tests\Manager;
+
+
+use Commercetools\Core\Model\Cart\Cart;
+use Commercetools\Core\Request\AbstractAction;
+use Commercetools\Symfony\CartBundle\Event\CartPostUpdateEvent;
+use Commercetools\Symfony\CartBundle\Event\CartUpdateEvent;
+use Commercetools\Symfony\CartBundle\Manager\CartManager;
+use Commercetools\Symfony\CartBundle\Model\CartUpdateBuilder;
+use Commercetools\Symfony\CartBundle\Model\Repository\CartRepository;
+use Prophecy\Argument;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class CartManagerTest extends \PHPUnit_Framework_TestCase
```

Comment:
Please use namespaced phpunit classes

Reply:
This is fixed at cart_bundle and is left unchanged here on purpose so that it will not cause conflicts in the future merge. Actually it was by mistake that this file was included in that specific commit :/